db/structure.sql

```diff
@@ -1119,6 +1126,202 @@ CREATE SEQUENCE public.country_properties_id_seq
 ALTER SEQUENCE public.country_properties_id_seq OWNED BY public.country_properties.id;
 
 
+--
+-- Name: dashboard_template_commodities; Type: TABLE; Schema: public; Owner: -
```

Comment:
I think we have stuff that doesn't belong here? One way to keep this file limited only to changes introduced in current branch is to 1. reset it to be a copy of base branch 2. `RAILS_ENV=test rake db:drop db:create db:structure:load` 3. `RAILS_ENV=test rake db:migrate`

Reply:
Yeah, I went down the lazy path, as always :P will do that and push my changes